app/scripts/modules/canary/acaTask/acaTaskStage.js

```diff
@@ -59,16 +59,28 @@ module.exports = angular.module('spinnaker.canary.acaTaskStage', [
         : $scope.stage.canary.watchers //if it is not an array it is probably a SpEL
       : '';
 
-    accountService.getUniqueAttributeForAllAccounts('aws', 'regions')
-      .then( (regions) => {
-        $scope.regions = regions.sort();
-      });
+    var applicationProviders = $scope.application.attributes.cloudProviders;
```

Comment:
avoid `var`: prefer `const` for values that won't change, or `let` for mutable variables